drivers/modem/modem_socket.c

```diff
@@ -267,37 +267,61 @@ int modem_socket_poll(struct modem_socket_config *cfg,
 		return -EINVAL;
 	}
 
+	k_sem_reset(&cfg->sem_poll);
+
 	for (i = 0; i < nfds; i++) {
 		sock = modem_socket_from_fd(cfg, fds[i].fd);
 		if (sock) {
 			/*
 			 * Handle user check for POLLOUT events:
-			 * we consider the socket to always be writeable.
+			 * we consider the socket to always be writable.
```

Comment:
This should be part of previous commit :)

Reply:
Which previous commit ? You would prefer another commit for this typo fix ?

Reply:
I thought this is a change to comment that you introduced in previous commit. But it is a comment already on master, so you can leave it as is :)